br/pkg/task/backup_ebs.go

```diff
@@ -185,6 +190,13 @@ func RunBackupEBS(c context.Context, g glue.Glue, cmdName string, cfg *BackupEBS
 			// TODO maybe we should consider remove snapshots already exists in a failure
 			return errors.Trace(err)
 		}
+
+		// resume scheduler after snapshot sent
+		if restoreE := restoreFunc(ctx); restoreE != nil {
+			log.Warn("failed to restore removed schedulers, you may need to restore them manually", zap.Error(restoreE))
+		}
```

Comment:
already done in feature branch, not sure why it's not reflected in freature branch